python/pyspark/pandas/tests/test_dataframe.py

```diff
@@ -6000,6 +6000,13 @@ def test_combine_first(self):
             expected_pdf = pd.DataFrame({"A": [None, 0], "B": [4.0, 1.0], "C": [3, 3]})
             self.assert_eq(expected_pdf, psdf1.combine_first(psdf2))
 
+    def test_multi_index_dtypes(self):
```

Comment:
Let's start following PySpark's convention now since we finished initial porting. Can you add a comment with JIRA prefix? See also https://spark.apache.org/contributing.html